### PR TITLE
feat: Analyses support

### PR DESCRIPTION
**Pull request by svipulrao98**
## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: new_lineage_panel/src/CustomNodes.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ export const NodeTypeIcon: FunctionComponent<{ nodeType: string }> = ({
     {nodeType === "model" && <ModelIcon />}
     {nodeType === "source" && <SourceIcon />}
     {nodeType === "exposure" && <ExposureIcon />}
+    {nodeType === "analysis" && <ExposureIcon />}
```
Comment (AdiGajbhiye) [inline new_lineage_panel/src/CustomNodes.tsx:83]: ask @sakibuiux for icon

**Comment by AdiGajbhiye** [comment]
in `TableDetails`, hide `sync with db` button

File: new_lineage_panel/src/CustomNodes.tsx [hunk 2]
```diff
@@ -319,12 +324,12 @@ export const TableNode: FunctionComponent<NodeProps> = ({ data }) => {
               View Details
             </div>
 
-            <div
+            {/* <div
```
Comment (AdiGajbhiye) [inline new_lineage_panel/src/CustomNodes.tsx:327]: why this change?

File: new_lineage_panel/src/CustomNodes.tsx [hunk 3]
```diff
@@ -80,6 +81,7 @@ export const NodeTypeIcon: FunctionComponent<{ nodeType: string }> = ({
     {nodeType === "model" && <ModelIcon />}
     {nodeType === "source" && <SourceIcon />}
     {nodeType === "exposure" && <ExposureIcon />}
+    {nodeType === "analysis" && <AnalysisIcon />}
```
Comment (saravmajestic) [inline new_lineage_panel/src/CustomNodes.tsx:84]: not in scope of this PR. but it is better to have defined enums for these strings

**Review by saravmajestic — COMMENTED** [review]
LGTM except the comment made by Aditya for the removed code

File: new_lineage_panel/src/CustomNodes.tsx (hunk 2 above)
Comment (svipulrao98) [reply new_lineage_panel/src/CustomNodes.tsx:327]: Have made changes for the 2nd ticket as well: [AI-1354](https://altimateai.atlassian.net/browse/AI-1354)

File: new_lineage_panel/src/CustomNodes.tsx [hunk 4]
```diff
@@ -272,6 +276,7 @@ export const TableNode: FunctionComponent<NodeProps> = ({ data }) => {
           e.stopPropagation();
           toggleTableSelection();
           highlightTable();
+          openFile(url);
```
Comment (AdiGajbhiye) [inline new_lineage_panel/src/CustomNodes.tsx:279]: remove this. this is getting in another PR.

Comment (AdiGajbhiye) [reply new_lineage_panel/src/CustomNodes.tsx:279]: @svipulrao98

**Review by mdesmet — APPROVED** [review]
